python/lsst/sconsUtils/tests.py

```diff
@@ -288,11 +288,11 @@ def runPythonTests(self, pyList):
         @printf "%s\\n" 'running global pytest... ';
         @({2} TRAVIS=1 {0} {1}); \
         export rc="$?"; \
-        if [[ $$rc == 0 ]]; then \
+        if [ "$$rc" -eq "0" ]; then \
```

Comment:
My reading of the documentation for `test` is that `-eq` is for integer checks and `=` is for string so I think we need `=` here.